tests/test_tutorial/test_events/test_tutorial002.py

```diff
@@ -3,6 +3,11 @@
 from fastapi.testclient import TestClient
 
 
+@pytest.fixture(autouse=True)
+def deactivate_blockbuster(blockbuster):
+    blockbuster.deactivate()
```

Comment:
starlette `Router`'s `shutdown` doesn't execute the shutdown event in a thread.
And `tutorial002` writes blockingly to the log file in its `shutdown_event`.
Here I deactivate blockbuster for the `tutorial002` test but maybe the tutorial could be changed to be fully async ? Or should starlette's `Router` defer the events execution to a thread ?